### PR TITLE
860306: Added get filtered row details in how to section of spreadsheet.

### PR DESCRIPTION
**Pull request by Vasanth-3519**
**Description:**
Added get filtered row details in how to section of spreadsheet documentation.

### CODE REVIEW

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        { width: 180 }, { width: 130 }, { width: 130 }, { width: 180 },
+        { width: 130 }, { width: 120 }
+    ]
+}];
+
+var spreadsheet = new ej.spreadsheet.Spreadsheet({
+    sheets: sheet,
+    allowFiltering: true,
+    created: function () {
+        // Applies cell and number formatting to specified range of the active sheet
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:15]: Remove the number formatting comments, just mention only cell formats

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        // Apply filter to the specified range.
+        spreadsheet.applyFilter(predicates, 'A1:C7');
+    }
+});
+
+// Render initialized Spreadsheet.
+spreadsheet.appendTo('#spreadsheet');
+
+document.getElementById("getFilterData").onclick = () => {
+    if (spreadsheet) {
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:33]: Remove the unwanted conditions

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+let columns: ColumnModel[] = [
+    { width: 180 }, { width: 130 }, { width: 130 }, { width: 180 },
+    { width: 130 }, { width: 120 }
+]
+
+let spreadsheet: Spreadsheet = new Spreadsheet({
+    sheets: [{ ranges: [{ dataSource: defaultData }], columns: columns }],
+    allowFiltering: true,
+    created: function () {
+        // Applies cell and number formatting to specified range of the active sheet
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts:15]: Modify the comments

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js (hunk 1 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:15]: Addressed and modified

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js (hunk 2 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:33]: Addressed and modified

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts (hunk 3 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts:15]: Addressed and modified

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    ranges: [{ dataSource: defaultData }],
+    columns: [
+        { width: 180 }, { width: 130 }, { width: 130 }, { width: 180 },
+        { width: 130 }, { width: 120 }
+    ]
+}];
+
+let spreadsheet = new ej.spreadsheet.Spreadsheet({
+    sheets: sheet,
+    allowFiltering: true,
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:12]: remove the allowFiltering property in code block

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+            operator: 'equal',
+            value: 'Pink',
+            matchCase: false
+        }];
+        // Apply filter to the specified range.
+        spreadsheet.applyFilter(predicates, 'A1:C7');
+    }
+});
+
+// Render initialized Spreadsheet.
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:28]: Remove the unwanted comment lines

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+import { defaultData } from './datasource.ts';
+
+let columns: ColumnModel[] = [
+    { width: 180 }, { width: 130 }, { width: 130 }, { width: 180 },
+    { width: 130 }, { width: 120 }
+]
+
+let spreadsheet: Spreadsheet = new Spreadsheet({
+    sheets: [{ ranges: [{ dataSource: defaultData }], columns: columns }],
+    allowFiltering: true,
```
Comment (sangeetha-priya-M) [inline ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts:13]: remove filtering api

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js (hunk 4 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:12]: removed.

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js (hunk 5 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.js:28]: removed

File: ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts (hunk 6 above)
Comment (Vasanth-3519) [reply ej2-javascript/code-snippet/spreadsheet/filter-cs2/index.ts:13]: removed

File: ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html [hunk 7]
```diff
@@ -43,6 +43,7 @@
 <li><a href="/ej2-javascript/spreadsheet/how-to/identify-the-context-menu-opened">Identify the context menu opened</a></li>
 <li><a href="/ej2-javascript/spreadsheet/how-to/save-and-open-spreadsheet-data-as-base64-string">Save and open Spreadsheet data as Base64 string</a></li>
 <li><a href="/ej2-javascript/spreadsheet/how-to/insert-sheet-programmatically-and-make-it-the-active-sheet">Insert a sheet programmatically and make it the active sheet</a></li>
+<li><a href="/ej2-javascript/spreadsheet/how-to/get-filtered-row-details">Get filtered row details</a></li>
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html:46]: file name : get-filtered-rows

Content: Get filtered rows

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Get the filtered row details in the ##Platform_Name## Spreadsheet control | Syncfusion
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:3]: Get the filtered rows in ##Platform_Name## Spreadsheet control | Syncfusion

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+description: Learn here all about how to get the filtered row details in Syncfusion ##Platform_Name## Spreadsheet control of Syncfusion Essential JS 2 and more.
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:4]: Learn here all about how to get the filtered rows in Syncfusion ##Platform_Name## Spreadsheet control of Syncfusion Essential JS 2 and more.

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+platform: ej2-javascript
+control: Spreadsheet
+publishingplatform: ##Platform_Name##
+documentation: ug
+domainurl: ##DomainURL##
+---
+
+## Get the filtered row details in ##Platform_Name## Spreadsheet control
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:12]: ## Get the filtered rows in ##Platform_Name## Spreadsheet control

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+Filtering helps you to view specific rows in the spreadsheet by hiding the other rows. You can use the [allowFiltering](https://ej2.syncfusion.com/documentation/api/spreadsheet/#allowfiltering) property to enable or disable filtering functionality. Using the [applyFilter](https://ej2.syncfusion.com/documentation/api/spreadsheet/#applyfilter) method you can be able to filter the data present. With the help of isFiltered property you can be able to identify the filtered row.
```
Comment (gobalakrishnans) [inline ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:14]: Filtering allows you to view specific rows in a spreadsheet by hiding the other rows. You can enable or disable filtering functionality using the allowFiltering property. You can also use the applyFilter method to filter the data. The filtered rows can be obtained by using the isFiltered property on each row.

**Review by gobalakrishnans — CHANGES_REQUESTED** [review]

File: ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html (hunk 7 above)
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/ej2-javascript-spreadsheet-toc.html:46]: Modified as instructed

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md (hunk 8 above)
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:3]: Changed.

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md (hunk 9 above)
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:4]: Changed.

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md (hunk 10 above)
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:12]: Changed.

File: ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md (hunk 11 above)
Comment (Vasanth-3519) [reply ej2-javascript/spreadsheet/how-to/get-filtered-row-details.md:14]: Modified the content as shared.

**Review by gobalakrishnans — APPROVED** [review]

**Pull request merged**